audit.json

```diff
@@ -46,5 +46,13 @@
  "100000_A Client Error response code was returned by the server_http://em-npa-aat.service.core-compute-aat.internal/api/document-tasks?offset=10\u0026pageNumber=10\u0026pageSize=10\u0026paged=true\u0026sort.sorted=true\u0026sort.unsorted=true\u0026unpaged=true_GET": "ignore",
  "100000_A Client Error response code was returned by the server_http://em-npa-aat.service.core-compute-aat.internal/api/markups_PUT": "ignore",
  "10021_X-Content-Type-Options Header Missing_http://em-npa-aat.service.core-compute-aat.internal/v2/api-docs_GET": "ignore",
- "10010_Cookie No HttpOnly Flag_http://em-npa-aat.service.core-compute-aat.internal/v2/api-docs_GET": "ignore"
+ "10010_Cookie No HttpOnly Flag_http://em-npa-aat.service.core-compute-aat.internal/v2/api-docs_GET": "ignore",
+ "10010_Cookie No HttpOnly Flag_http://em-npa-aat.service.core-compute-aat.internal/api/markups/documentId?offset=10\u0026paged=true\u0026pageNumber=10\u0026pageSize=10\u0026sort.sorted=true\u0026sort.unsorted=true\u0026unpaged=true_GET":"ignore",
+ "10010_Cookie No HttpOnly Flag_http://em-npa-aat.service.core-compute-aat.internal/api/document-tasks?offset=10\u0026paged=true\u0026pageNumber=10\u0026pageSize=10\u0026sort.sorted=true\u0026sort.unsorted=true\u0026unpaged=true_GET":"ignore",
+ "10054_Cookie Without SameSite Attribute_http://em-npa-aat.service.core-compute-aat.internal/api/document-tasks?offset=10\u0026paged=true\u0026pageNumber=10\u0026pageSize=10\u0026sort.sorted=true\u0026sort.unsorted=true\u0026unpaged=true_GET":"ignore",
+ "10054_Cookie Without SameSite Attribute_http://em-npa-aat.service.core-compute-aat.internal/api/markups/documentId?offset=10\u0026paged=true\u0026pageNumber=10\u0026pageSize=10\u0026sort.sorted=true\u0026sort.unsorted=true\u0026unpaged=true_GET":"ignore",
+ "90033_Loosely Scoped Cookie_http://em-npa-aat.service.core-compute-aat.internal/api/markups/documentId?offset=10\u0026paged=true\u0026pageNumber=10\u0026pageSize=10\u0026sort.sorted=true\u0026sort.unsorted=true\u0026unpaged=true_GET":"ignore",
+ "90033_Loosely Scoped Cookie_http://em-npa-aat.service.core-compute-aat.internal/api/document-tasks?offset=10\u0026paged=true\u0026pageNumber=10\u0026pageSize=10\u0026sort.sorted=true\u0026sort.unsorted=true\u0026unpaged=true_GET":"ignore",
+ "100000_A Client Error response code was returned by the server_http://em-npa-aat.service.core-compute-aat.internal/api/document-tasks?offset=10\u0026paged=true\u0026pageNumber=10\u0026pageSize=10\u0026sort.sorted=true\u0026sort.unsorted=true\u0026unpaged=true_GET":"ignore",
```

Comment:
why are these client errors ignored?

Reply:
> why are these client errors ignored?



> why are these client errors ignored?

these are just informational errors and this kind of errors were ignored previously as well.